common/app/model/content.scala

```diff
@@ -423,7 +424,7 @@ final case class Article (
 
   val lightbox = GenericLightbox(content.elements, content.fields, content.trail, lightboxProperties)
 
-  val isLiveBlog: Boolean = content.tags.isLiveBlog
+  val isLiveBlog: Boolean = content.tags.isLiveBlog && content.fields.blocks.nonEmpty
```

Comment:
now we only get the blocks if there is live in the url, and if there are no blocks it assumes it's an article not a live blog.  This does cause some formatting issues for old live blogs without live in the url, but Matt W is happy enough with that since they don't get much traffic and they will be fixed anyway once we finish the migration and switch to using blocks for everything.